src/cf-buttons/usage.md

```diff
@@ -361,6 +362,26 @@ For accessibility reasons, use the semantic `<button>` instead of a link when po
 <input type="submit" value="Input Tag" class="a-btn a-btn__super active">
 ```
 
+### Full-width buttons on x-small screens
+
+_Reduce screen size to see these in action_
```

Comment:
Yeah, I didn't mean for you to work on it here. Just wanted to document the idea to consider for the future.